<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserGroupsDialog</class>
 <widget class="QDialog" name="UserGroupsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="5" colspan="2">
    <widget class="QListWidget" name="groupsListWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QLineEdit" name="descrLineEdit"/>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QLabel" name="colorLabel">
     <property name="text">
      <string>#000000</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Members:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="4">
    <widget class="QPlainTextEdit" name="membersPlainTextEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="newPushButton">
     <property name="text">
      <string>New</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/trolltech/filenew.png</normaloff>:/icons/trolltech/filenew.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QPushButton" name="deletePushButton">
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/editdelete.png</normaloff>:/icons/editdelete.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="savePushButton">
     <property name="text">
      <string>Save</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/trolltech/filesave.png</normaloff>:/icons/trolltech/filesave.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>96</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="5">
    <widget class="QPushButton" name="closePushButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closePushButton</sender>
   <signal>clicked()</signal>
   <receiver>UserGroupsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newPushButton</sender>
   <signal>clicked()</signal>
   <receiver>UserGroupsDialog</receiver>
   <slot>onNewClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deletePushButton</sender>
   <signal>clicked()</signal>
   <receiver>UserGroupsDialog</receiver>
   <slot>onDeleteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>savePushButton</sender>
   <signal>clicked()</signal>
   <receiver>UserGroupsDialog</receiver>
   <slot>onSaveClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorLabel</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>UserGroupsDialog</receiver>
   <slot>onColorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onNewClicked()</slot>
  <slot>onDeleteClicked()</slot>
  <slot>onSaveClicked()</slot>
  <slot>onColorClicked()</slot>
  <slot>onCurrentGroupModified()</slot>
 </slots>
</ui>
